cmd/limactl/copy.go

```diff
@@ -65,6 +65,48 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		verbose = true
 	}
 
+	cpTool := rsync
+	arg0, err := exec.LookPath(string(cpTool))
+	if err != nil {
+		arg0, err = exec.LookPath(string(cpTool))
+		if err != nil {
+			return err
+		}
+	}
+	logrus.Infof("using copy tool %q", arg0)
+
+	var sshArgs, toolArgs []string
+
+	switch cpTool {
+	case scp:
+		sshArgs, toolArgs, err = useScp(args, verbose, recursive)
+		if err != nil {
+			return err
+		}
+	case rsync:
+		toolArgs, err = useRsync(args, verbose, recursive)
+		if err != nil {
+			return err
+		}
+	default:
+		return fmt.Errorf("invalid copy tool %q", cpTool)
+	}
+
+	sshCmd := exec.Command(arg0, append(sshArgs, toolArgs...)...)
```

Comment:
appending empty sshArgs to rsync will work but is very confusing.

Also I'm not sure why we ned to handle arg0 separately. The functions can return a complete csp or rsync command ([]string) and here we can create the command like this:

    copyCmd = exec.Command(command[0], command[1:]...)

Or, return `*exec.Command` from the functions instead of lists.